lib/components/opaque_image_view.js

```diff
@@ -16,10 +16,18 @@ const GeminiHost = 'd7hftxdivxxvm.cloudfront.net';
 const ImageQuality = 85;
 
 export default class OpaqueImageView extends React.Component {
+  state: {
+    aspectRatio: ?number,
+    width?: number | string,
+    height?: number | string
+  };
```

Comment:
to unpack this, if it's the first time you've seen this stuff.

aspectRatio is an optional number,
width is an optional key, that can be a number or a string 
height is an optional key, that can be a number or a string 

the number/string stuff is weird, because we want to pass width and height back inside a string. Maybe there's a better way to say print this number as a string that I've not found in flow.